### PR TITLE
Update web app styling and pages

### PR DESCRIPTION
**Pull request by GiuliaTeggi**
- Add landing pages for each event (birth, health, death etc) with 'coming soon' actions
- Update styling 
- Move some components to `components` folders

In a following pr I'll add feature flags for each menu item so that they can be hidden or shown.

Screenshot 2024-03-22 at 17 44 05

Screenshot 2024-03-25 at 09 32 36

### CODE REVIEW

File: apps/web/app/[locale]/Header.tsx [hunk 1]
```diff
@@ -1,149 +0,0 @@
-import Link from "next/link";
```
Comment (GiuliaTeggi) [inline apps/web/app/[locale]/Header.tsx:1]: This component was just moved and updated, not deleted

File: apps/web/app/utils/routes.ts [hunk 2]
```diff
@@ -151,7 +151,75 @@ export const driving = {
     },
   },
 };
-export const employment = { slug: "employment" };
+export const employment = {
```
Comment (GiuliaTeggi) [inline apps/web/app/utils/routes.ts:154]: This is simply copied from #27 - which should be merged before this pr.

**Review by Thurfjell — APPROVED** [review]
Lots of this is gonna need to be tweaked when we start consider workflows potentially coming from the messaging system. 

Which workflows should be static and which are more dynamic? 🤷 

This is good stuff regardless. Strong work.

**Comment by GiuliaTeggi** [comment]
> Lots of this is gonna need to be tweaked when we start consider workflows potentially coming from the messaging system.
> 
> Which workflows should be static and which are more dynamic? 🤷
> 
> This is good stuff regardless. Strong work.

Yes absolutely, this is really only to have some landing pages to show in the demo I guess

**Pull request merged**